cnames_active.js

```diff
@@ -900,6 +900,7 @@ var cnames_active = {
   "easytrivia-guide": "turtlepaw.github.io/trivia-docs",
   "eatery-nod-w": "kevinast.github.io/eatery-nod-w",
   "eazyd": "itzarman09.github.io/discord-eazy",
+  "ebay-wrapper": "tradebuddyhq.github.io/ebay-wrapper.js.org:,
```

Comment:
```suggestion
  "ebay-wrapper": "tradebuddyhq.github.io/ebay-wrapper.js.org",
```
You're missing a quote